### ISSUE TITLE
UI: Allow graph zoom further than 1 day when in hourly view

### ISSUE THREAD

**Issue by zachmullen** [XDATA-Year-3/geoapp#146]
Also, several users expected that clicking on an hour would filter down to that hour, much like it works for clicking on a single day.

**Comment by manthey** [comment]
Did analysts find the hour view useful in general?  Not many switched the time grouping when I was there.

**Comment by zachmullen** [comment]
Two or three used it to try and find the start of the marathon (not the easiest way, but they tried nonetheless). Were you thinking of removing hourly resolution altogether?

**Comment by manthey** [comment]
No.  I could see it being useful in some of the real-time data collection processes.  I think allowing zooming in further make sense, but only when at least one graph is set to hourly.

**Comment by jamiesnape** [comment]
I can a user who really liked the different resolutions and used the monthly one extensively. No one used the hourly, but I think that was the nature of the tasks. In general, the zoom on the graphs was the most difficult for users to understand, although they all seemed to struggle in different ways.
